testing/metaopt/test_automl_sequential_wrapper.py

```diff
@@ -40,3 +37,43 @@ def test_automl_sequential_wrapper(tmpdir):
     # Then
     actual_mse = ((predicted_outputs - expected_outputs) ** 2).mean()
     assert actual_mse < mse_before
+
+
+def test_automl_sequential_wrapper_with_validation_split_wrapper(tmpdir):
+    # Given
+    data_inputs = np.array(range(100))
+    expected_outputs = np.array(range(100, 200))
+
+    hyperparameter_space = HyperparameterSpace({
+        'multiplication_1__multiply_by': RandInt(1, 3),
+        'multiplication_2__multiply_by': RandInt(1, 3),
+        'multiplication_3__multiply_by': RandInt(1, 3),
+    })
+
+    pipeline = Pipeline([
+        ('multiplication_1', MultiplyByN()),
+        ('multiplication_2', MultiplyByN()),
+        ('multiplication_3', MultiplyByN())
+    ], cache_folder=tmpdir).set_hyperparams_space(hyperparameter_space)
+
+    random_search = RandomSearch(
+        pipeline,
+        hyperparams_repository=HyperparamsJSONRepository(tmpdir),
+        validation_technique=ValidationSplitWrapper(
+            test_size=0.2,
+            scoring_function=mean_squared_error,
+            run_validation_split_in_test_mode=False
+        ),
+        n_iters=100,
+        higher_score_is_better=False
+    )
+
+    # When
+    mse_before = ((data_inputs - expected_outputs) ** 2).mean()
```

Comment:
is mse_before the good name for this variable? This is confusing

Reply:
Note that the present test looks like it would be a good documentation example for the gallery.